beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -651,3 +651,221 @@ func IsBetterUpdate(newUpdate, oldUpdate *v2.LightClientUpdate) (bool, error) {
 	}
 	return newUpdate.SignatureSlot < oldUpdate.SignatureSlot, nil
 }
+
+func BlockToLightClientHeaderAltair(block interfaces.ReadOnlySignedBeaconBlock) (*structs.LightClientHeader, error) {
+	parentRootArray := block.Block().ParentRoot()
+	parentRoot := parentRootArray[:]
+
+	stateRootArray := block.Block().StateRoot()
+	stateRoot := stateRootArray[:]
```

Comment:
same here why are we copying it and then allocating it.

Reply:
> same here why are we copying it and then allocating it.

nice catch, thanks